### ISSUE TITLE
itunes

### ISSUE THREAD

**Issue by despoinamusica39** [slayrau/itunes-search#1]
hello there...happy new year!could you please complete the project for itunes?for download it from your github?I have no idea from programming!!!I will preciate it so much!!!

**Comment by slayrau** [comment]
Hello! I am working on it, it will take a while.

**Comment by despoinamusica39** [comment]
helloooooooo from Greece!!!thanks so muchhhhhhhhhhhh where answer me!tell me please more less 2-3 hours it will be ready?

**Comment by despoinamusica39** [comment]
and please I Am begging you could you make it increase unlimited songs and albums?it would be great!!!there is way!look I found a way to query more records. There is a parameter called offset, which is not mentioned in API documentation.

https://itunes.apple.com/search?term=a&offset=25&limit=25

**Comment by slayrau** [comment]
> tell me please more less 2-3 hours it will be ready?
No, it’s not. I plan on doing more things like an inner page for each content type. The inner movie page will show everything about the filmed movie, for example, description, actors, genre, also I want to make a video player to display the movie trailer. The inner music page will display information about the album, you can play previews of songs, etc. All this can take more than one week.

>could you make it increase unlimited songs and albums?
Yes, I'm working on it right now. As far as I know, there is a limit of 200 entries, but I'll double-check that later.

**Comment by despoinamusica39** [comment]
woowwwwww thanks a lot my friend!with this you will can increase ulimited results!!!

**Comment by despoinamusica39** [comment]
hello there!please can you make one web for your project?
